### ISSUE TITLE
482 Loop Detected response is generated to INVITE that starts a new transaction

### ISSUE THREAD

**Issue by juha-h** [versatica/JsSIP#617]
When JsSIP receives a new INVITE request after the transaction created by the previous one is not anymore ongoing, it generates 482 Loop Detected response instead of creating a new transaction.

The issue could be resolved by adding to sanityCheck.js rfc3261_8_2_2_2() function's merged request test an additional condition that skips the test for transactions that are  not anymore ongoing:
```
8.2.2.2 Merged Requests

   If the request has no tag in the To header field, the UAS core MUST
   check the request against ongoing transactions.  If the From tag,
   Call-ID, and CSeq exactly match those associated with an ongoing
   transaction, but the request does not match that transaction (based
   on the matching rules in Section 17.2.3), the UAS core SHOULD
   generate a 482 (Loop Detected) response and pass it to the server
   transaction.
```
In case of INVITE transaction, the merged request test should be skipped if ACK has already been received for the transaction.

**Comment by juha-h** [comment]
I added one more test to rfc3261_8_2_2_2() if statement:
```
          tr = ua._transactions.ist[transaction];
          if (tr.request.state < Transactions.C.STATUS_TERMINATED &&
              tr.request.from_tag === fromTag &&
              tr.request.call_id === call_id &&
              tr.request.cseq === cseq)
          {
            reply(482);

            return false;
          }
```
After that the second INVITE was accepted.  But that is not enough. JsSIP should not start first ringing an after that check if request is acceptable.

**Comment by jmillan** [comment]
fixed in 3.4.3.
